frontend/templates/troubleshooting/index.tsx

```diff
@@ -901,14 +901,30 @@ function Conclusion({ conclusion: conclusions }: { conclusion: Section[] }) {
 
 function AnswersCTA({ answersUrl }: { answersUrl: string }) {
    return (
-      <Alert p={3} mt={4}>
-         <AlertIcon color="gray.500" />
-         <chakra.span pr={3} mr="auto">
-            Haven&apos;t found the solution to your problem?
-         </chakra.span>
-         <Button href={answersUrl} as="a" colorScheme="brand">
-            Browse our forum
-         </Button>
+      <Alert p={3} fontSize="sm">
+         <AlertIcon
+            color="gray.500"
+            alignSelf={{ base: 'start', sm: 'center' }}
+         />
+         <Flex
+            align="center"
+            flex="auto"
+            justify="space-between"
+            sx={{
+               '@media (max-width: 375px)': {
```

Comment:
The breakpoint here seems like something we'd rather pull from a constant, but I'm also good with this if it's basically experimentally determined.

Reply:
I've been avoiding using breakpoints due to the SSR issues. It's a gotcha I'd rather deal with upfront.